### ISSUE TITLE
[macos catalina] build failures

### ISSUE THREAD

**Issue by dnalborczyk** [denoland/deno#3440]
os: macOS Catalina 10.15.1

master @ https://github.com/denoland/deno/commit/32937251315493ef2c3b42dd29340e8a34501aa4

I followed the `build from source` instructions from: https://deno.land/std/manual.md#build-from-source

```
Failed to fetch file gs://chromium-clang-format/025ca7c75f37ef4a40f3a67d81ddd11d7d0cdb9b for /Users/daniel/dev/github-forks/deno/core/libdeno/buildtools/mac/clang-format. [Err: /Users/daniel/dev/github-forks/deno/third_party/depot_tools/external_bin/gsutil/gsutil_4.28/gsutil/third_party/boto/boto/pyami/config.py:69: UserWarning: Unable to load AWS_CREDENTIAL_FILE ()
  warnings.warn('Unable to load AWS_CREDENTIAL_FILE (%s)' % full_path)
]
thread 'main' panicked at 'assertion failed: status.success()', core/build.rs:94:9
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

I was able to create a successful build on an Ubuntu 18.04 docker image.

**Comment by dnalborczyk** [comment]
just found: https://github.com/denoland/deno/issues/3153#issuecomment-544673630 [This comment was marked as outdated]

commenting out `third_party.download_clang_format()` in `/tools/setup.py` fixed it for me. third_party.download_sccache() doesn't exist anymore.

https://github.com/denoland/deno/blob/32937251315493ef2c3b42dd29340e8a34501aa4/tools/setup.py#L30

**Comment by kevinkassimo** [comment]
@dnalborczyk Your `cargo test` failure looks a lot like you have some other server running on ports that the tests are using.

**Comment by dnalborczyk** [comment]
thank you @kevinkassimo , that fixed the test run!

it seems a `previous test run` might have kept the test server hanging around.

https://github.com/denoland/deno/blob/32937251315493ef2c3b42dd29340e8a34501aa4/tools/http_server.py#L16

```
lsof -nP -i4TCP:4545 | grep LISTEN 
Python  45071 daniel    6u  IPv6 0x4______________      0t0  TCP *:4545 (LISTEN)
```

**Comment by chrmoritz** [comment]
Looks like the fix floated over `third_party/depot_tools` in https://github.com/denoland/deno_third_party/commit/818d33a13b06c4863b0ddc20e89d35715c35b37c got accidentally reverted in the v8  7.9.317.12 upgrade in https://github.com/denoland/deno_third_party/commit/cab4821cd937dd3c927430f0f0e05a38af1b84a0#diff-452661d791c32d03de51b6ef4c687bf1L47-R47. Reapplying the first should fix running `.tools/setup.py` on Catalina again.

**Comment by chrmoritz** [comment]
This should hopefully stay fixed after #3441 (the `depot_tools` to the version used in v8 8.0 ) gets merged.

**Comment by dnalborczyk** [comment]
thank you @chrmoritz !! I can confirm: https://github.com/denoland/deno/pull/3441 fixed it!
